tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+- name: apt-get update && dist-upgrade
+  apt: upgrade=dist update_cache=yes cache_valid_time=18000
+
+- name: Check if packages need to be autoremoved
+  command: apt-get --dry-run autoremove
+  register: check_autoremove
+  changed_when: False
+
+- name: Autoremove unused packages
+  command: apt-get -y autoremove
+  when: "'packages will be REMOVED' in check_autoremove.stdout"
+
+- name: Install prerequisite packages
+  apt: pkg={{ item }} state=installed update_cache=yes cache_valid_time=18000
+  with_items: "{{ ROLE_INDEPENENT_USEFULL_PACKAGES }}"
+
+- name: configure unattened upgrades
+  template: src=unattended-upgrades dest=/etc/apt/apt.conf.d/50unattended-upgrades
```

Comment:
[Contents of this file](https://github.com/geerlingguy/ansible-role-security/blob/master/templates/50unattended-upgrades.j2) in the security role is as follows: 

```
Unattended-Upgrade::Automatic-Reboot "false";

Unattended-Upgrade::Allowed-Origins {
        "${distro_id} ${distro_codename}-security";
//      "${distro_id} ${distro_codename}-updates";
};
```

You need to add following lines to get an e-mail on failure: 

```
Unattended-Upgrade::Mail "root";
Unattended-Upgrade::MailOnlyOnError "true"; 
```